### ISSUE TITLE
document that hooks don't act in a non-DVC branch

### ISSUE THREAD

**Issue by gthb** [iterative/dvc.org#506]
Document that:

* (once https://github.com/iterative/dvc/pull/2338 is merged) the hooks installed by `dvc install` only act in a “dvc-enabled” branch, meaning one that has tracked files under `.dvc`
* initializing dvc on a branch doesn't affect other branches until the change gets merged to those branches. (So you can safely try it out in a branch before merging to master, for instance.)

Prompted by https://github.com/iterative/dvc/issues/2208

**Comment by jorgeorpinel** [comment]
I think this is intuitive, so probably no need to make a note about it in docs. Thanks!

**Comment by shcheklein** [comment]
@jorgeorpinel It looks worth mentioning this. In the checkout hook description.

**Comment by jorgeorpinel** [comment]
Hmmm. OK, added the note in https://github.com/iterative/dvc.org/pull/931/commits/02a13f3de2b507fd7c80c2f9faea7f2c8867ae6d but to me it seems like a very corner case, not sure it's worth documenting. People who only have DVC in certain branches of a repo? Monorepo projects I guess.

**Comment by shcheklein** [comment]
@jorgeorpinel no, it's simpler. You introduce DVC and it's not yet merged into all branches properly.

**Comment by jorgeorpinel** [comment]
I see, thanks. Please notice https://github.com/iterative/dvc.org/pull/931#pullrequestreview-345502870 when you have a chance.

**Comment by jorgeorpinel** [comment]
^ Never mind, I took a decision. You'll see it when you review the PR. Thanks
